sailthru_content/services/sailthru_translation_service.py

```diff
@@ -0,0 +1,275 @@
+import datetime
+import logging
+
+
+class SailthruTranslationService(object):
+    def __init__(self, data_service, lms_root, fixups=None):
+        self.data_service = data_service
+        self.lms_root = lms_root
+        self.fixups = fixups
+
+    def _convert_date(self, iso_date):
+        """ Convert date from ISO 8601 (e.g. 2016-04-15T20:35:11.424818Z) to Sailthru format
+        """
+        if iso_date is None:
+            return None
+        try:
+            if '.' in iso_date:
+                return datetime.datetime.strptime(iso_date, "%Y-%m-%dT%H:%M:%S.%fZ").strftime("%Y-%m-%d")
+            return datetime.datetime.strptime(iso_date, "%Y-%m-%dT%H:%M:%SZ").strftime("%Y-%m-%d")
+        except ValueError:
+            return None
+
+    def _convert_datetime(self, iso_date):
+        """ Convert date from ISO 8601 (e.g. 2016-04-15T20:35:11.424818Z) to Sailthru format
+        """
+        if iso_date is None:
+            return None
+        try:
+            if '.' in iso_date:
+                return datetime.datetime.strptime(iso_date, "%Y-%m-%dT%H:%M:%S.%fZ").strftime("%Y-%m-%d %H:%M:%S +0000")
+            return datetime.datetime.strptime(iso_date, "%Y-%m-%dT%H:%M:%SZ").strftime("%Y-%m-%d %H:%M:%S +0000")
+        except ValueError:
+            return None
+
+    def _convert_tag(self, tagtype, tag):
+        """ Convert string to a valid sailthru tag and add type- to the front if specified
+        """
+        if tag:
+            resp = tag.replace(' & ', '-').replace(',', '').replace('.', '').replace(' ', '-').replace('--', '-')
+            if tagtype:
+                resp = tagtype + '-' + resp
+            return resp
+        return ''
+
+    def _create_course_vars(self, course, course_run, url, site_name, program_dictionary=None):
+        """ Generate 'vars' section of Sailthru data
```

Comment:
Nitpick: keep docstring format consistent with our other repos (i.e., put the closing quote on the same line, or put the docstring on its own line).